_includes/quiz-funnel/1-intro.html

```diff
@@ -0,0 +1,72 @@
+<section class="primary claim container">
+  <h1 class="display-6">
+    Werde <strong>in 6 Monaten zum Senior-Developer!</strong>
+    <br />
+    Eine <strong>Weiterbildung</strong> für Web- und App-Entwickler:innen auf Junior-Level.
+  </h1>
+</section>
+<div class="container">
+  <div class="card overflow-hidden">
+    <div class="row g-0">
+      <div class="image col-md-6 d-flex justify-content-center">
+        {% picture academy_4.png --alt unser Angebot %}
+      </div>
+      <div class="col-md-6 story-position">
+        <div class="card-body p-4 px-lg-5">
+          <h3 class="card-title h5 mb-1 text-center">
+            Unser Angebot
+          </h3>
+          <ul class="list-group list-group-flush px-4 px-lg-5">
+            <li class="list-group-item px-0">
+              <strong>Weiterbildung</strong> für Software Developer mit min. <strong>6 Monate Berufserfahrung</strong>
+            </li>
+            <li class="list-group-item px-0">
+              Inhaltlich zielt der Kurs auf <strong>Full Stack Development</strong> ab

```

Comment:
Wir sprechen eigentlich immer von "Programm" und nicht von "Kurs", weil das umfassender ist.
```suggestion
              In diesem Programm lernst du alles, was du über <strong>Full Stack Development</strong> wissen musst.

```

Reply:
ALLE Seiten des Quiz-Funnels befinden sich im Verzeichnis quiz-funnel im Hauptverzeichnis. Es gibt keine Dateien mehr im _includes verzeichnis und auch keine md-Files mehr, eventuell hast du die méinmal gepullt und wieder gepusht?